Description: Relax required internal build depends  for Plasma
 This avoids needing to build all Plasma sources at the new version
 where many do not have any code changes apart from a version bump.
Forwarded: not-needed
Author: Rik Mills <rikmills@kde.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@
 
 project(xdg-desktop-portal-kde VERSION ${PROJECT_VERSION})
 
-set(PROJECT_DEP_VERSION "6.6.6")
+set(PROJECT_DEP_VERSION "6.6.4")
 set(QT_MIN_VERSION "6.10.0")
 set(KF6_MIN_VERSION "6.22.0")
 set(KDE_COMPILERSETTINGS_LEVEL "6.10.0")
